server/plugins/messageStorage/sqlite.ts

```diff
@@ -230,15 +230,10 @@ class SqliteMessageStorage implements ISqliteMessageStorage {
 		params.push(query.offset);
 
 		const rows = await this.serialize_fetchall(select, ...params);
-		const response: SearchResponse = {
-			searchTerm: query.searchTerm,
-			target: query.channelName,
```

Comment:
we renamed this for absolutely no reason...
Hence I propose that we get rid of the mental gymnastics involved and just call it what it really is

Reply:
Pretty sure I copied this all over during the typescript transition. Feel free to improve it